iChef-WebUi/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { HEADER_LINKS } from 'src/app/constants/header';
+import { BUTTON, LANDING_PAGE } from 'src/app/constants/texts';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
+export class HeaderComponent {
+
+    readonly homeLink = HEADER_LINKS.HOME;
+    readonly recipesLink = HEADER_LINKS.RECIPES;
+    readonly loginBtn = BUTTON.LOGIN;
+    readonly registerBtn = BUTTON.REGISTER;
+    readonly appTitle = LANDING_PAGE.TITLE;
+
+  headerLinks = new Map<string, boolean>([
```

Comment:
Why do we need this variable? If it is to keep track of what page the user is on so that we can style the button then I don't think we need it since angular already can help you with that. Angular can add a css class to an element if that element is a link to the active route.

Reply:
It seemed more simple this way to access each link and reset the links to their initial class (black text) when switching pages. We will have more links if the user is logged in ('profile' and 'settings' maybe). Tho I think this can be improved by looking at the current route and deciding by that. I will look into this next time.

Reply:
The thing is right now this variable seems to not be used anywhere. The value is set but it is never read. Rather than making possibly long lasting decisions about something that we will only need in the future I would not add this code right now and when we need it we will add here.

Reply:
Nevermind just saw that his is used in html. My bad, I will close the conversation